core/services/ping/ping360_driver.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import shlex
+from pathlib import Path
+from shutil import which
+from subprocess import Popen
+
+import serial
+from brping import PingDevice
+from brping.definitions import COMMON_DEVICE_INFORMATION
+
+from pingdriver import PingDriver
+from pingutils import PingDeviceDescriptor
+
+BAUDRATES = [
```

Comment:
how relevant is ABR for ping1d? does it make anything faster/better?

Reply:
Is not necessary, but the latest and future firmwares supports, I'm saying this to unify both serial integrations in a single class to avoid duplicate/unnecessary in both classes and share the same know behavior when using via serial.